pulp_deb/app/tasks/synchronizing.py

```diff
@@ -75,12 +75,31 @@ class NoReleaseFile(Exception):
     Exception to signal, that no file representing a release is present.
     """
 
-    def __init__(self, distribution, *args, **kwargs):
+    def __init__(self, url, *args, **kwargs):
         """
         Exception to signal, that no file representing a release is present.
         """
         super().__init__(
-            "No valid Release file found for '{}'.".format(distribution), *args, **kwargs
+            "Could not find a Release file at '{}', try checking the 'url' and "
+            "'distributions' option on your remote".format(url),
+            *args,
+            **kwargs,
+        )
+
+
+class NoValidGPG(Exception):
```

Comment:
Nitpick:
```suggestion
class NoValidSignatureForKey(Exception):
```
I feel like this is a better name, since we don't know if the problem is with the GPG key or with the signature.

Reply:
Changed the name.